sockssrv.c

```diff
@@ -291,6 +297,27 @@ int main(int argc, char** argv) {
 		return usage();
 	}
 	signal(SIGPIPE, SIG_IGN);
+
+	if(inetd) {
+		struct client c;
+		c.fd = STDIN_FILENO;
+		void *peer = SOCKADDR_UNION_ADDRESS(&c.addr);
+		socklen_t peerlen = SOCKADDR_UNION_LENGTH(&c.addr);
+
+		if (getpeername(c.fd, (struct sockaddr *)peer, &peerlen) < 0) {
+			dprintf(2, "error: inetd getpeername()\n");
+			return 1;
+		}
+
+		int fd = connect_target(&c);
+		if(fd != -1) {
+			copyloop(c.fd, fd);
+			close(fd);
+		}
```

Comment:
what happens in case of error ?

Reply:
Hi @rofl0r ,

Yes! Thank you for noticing it. Now, it returns an error exit code. I prefer to not print a message, because in the Inet mode it doesn't have sense. You agree?

Reply:
i never used inetd so i dont really know. i guess it's ok now...
i'll read up about inetd next couple days and revise this PR. thanks for the clean work!

Reply:
Hi @rofl0r ,

You can test it in any standard Unix like system: using `inted` or `xinted` daemons. It's very simple. But in fact, you can check in another way: using `netcat` (or `ncat` or `nc`) in listening mode. Because the idea is simple: it reads from the STDIN for the _incoming_ connection.

Regards.